samza-api/src/main/java/org/apache/samza/util/RateLimiter.java

```diff
@@ -110,11 +101,8 @@
   Map<String, Integer> acquire(Map<String, Integer> tagToCreditMap, long timeout, TimeUnit unit);
 
   /**
-   * Attempt to acquire the provided number of credits for a number of tags, returns immediately number of
-   * credits acquired.
-   *
-   * @param tagToCreditMap a map of requested number of credits keyed by tag
-   * @return a map of number of credits acquired keyed by tag
+   * Get the entire set of tags for which we have configured credits for rate limiting.
+   * @return set of supported tags
    */
-  Map<String, Integer> tryAcquire(Map<String, Integer> tagToCreditMap);
+  Set<String> getSupportedTags();
```

Comment:
it necessary becaues the remote table needs to figure out whether we are rate limiting read or write or both. Since we accept user's rate limiter instance where they can be in either one of the three cases above. In the implementation of TaggedRL, if caller tries to call acquire for a tag which has not been configured it'll throw. 

Since there is no way to figure out which tags have been configured by the passed-in ratelimiter instance, there is no way to know which operation needs rate limiting. 

So adding this method help us in this regard. It's just that I forgot to use it in RemoteTable. I'll update the commit. 

Please let me know if this makes sense. Good catch btw!